packages/postcss/src/esm.ts

```diff
@@ -105,12 +104,12 @@ export function createPlugin(options: UnoPostcssPluginOptions) {
     const globs = uno.config.content?.filesystem ?? defaultFilesystemGlobs
     const plainContent = uno.config.content?.inline ?? []
 
-    const entries = await fg(isScanTarget ? globs : from, {
+    const entries = await glob(isScanTarget ? globs : [from], {
```

Comment:
cannot add `then` and find stats using `node:fs/promises`, we should add small cache to avoid process unchanged files

Reply:
i'm not sure i understand, do you mean using `node:fs/promises`' `stats` to use `fileMap` the same way it was used before?

Reply:
yes, if the logic was wrong just fix it